jujugui/static/gui/src/app/components/status/status.js

```diff
@@ -177,54 +177,27 @@ class Status extends React.Component {
   }
 
   render() {
+
     return (
       <SharedStatus
-        entities={{
-          applications: this.props.db.services.filter(app => !app.get('pending')),
-          machines: this.props.db.machines.filter(mach => mach.id.indexOf('new') !== 0),
-          model: this.props.model,
-          relations: this.props.db.relations.filter(rel => !rel.get('pending')),
-          units: utils.getRealUnits(this.props.db.units),
-          remoteApplications: this.props.db.remoteServices.toArray()
-        }}
         generateApplicationOnClick={this._generateApplicationOnClick.bind(this)}
         generateApplicationURL={this._generateApplicationURL.bind(this)}
         generateCharmURL={this._generateCharmURL.bind(this)}
         generateMachineOnClick={this._generateMachineOnClick.bind(this)}
         generateMachineURL={this._generateMachineURL.bind(this)}
         generateUnitOnClick={this._generateUnitOnClick.bind(this)}
         generateUnitURL={this._generateUnitURL.bind(this)}
+        model={this.props.model}
         navigateToApplication={this._navigateToApplication.bind(this)}
         navigateToCharm={this._navigateToCharm.bind(this)}
-        navigateToMachine={this._navigateToMachine.bind(this)} />
+        navigateToMachine={this._navigateToMachine.bind(this)}
+        valueStore={this.props.valueStore} />
     );
   }
 };
 
 Status.propTypes = {
   changeState: PropTypes.func.isRequired,
-  db: shapeup.shape({
-    machines: shapeup.shape({
-      filter: PropTypes.func.isRequired,
-      toArray: PropTypes.func.isRequired
-    }).isRequired,
-    relations: shapeup.shape({
-      filter: PropTypes.func.isRequired
-    }).isRequired,
-    remoteServices: shapeup.shape({
-      map: PropTypes.func.isRequired,
-      toArray: PropTypes.func.isRequired
-    }).isRequired,
-    services: shapeup.shape({
-      filter: PropTypes.func.isRequired,
-      getById: PropTypes.func.isRequired,
-      toArray: PropTypes.func.isRequired
-    }).isRequired,
-    units: shapeup.shape({
-      filter: PropTypes.func.isRequired,
-      toArray: PropTypes.func.isRequired
-    }).isRequired
-  }).frozen.isRequired,
   generatePath: PropTypes.func.isRequired,
   model: shapeup.shape({
```

Comment:
maybe this shape should also go into the valuestore prop types.

Reply:
Maybe, it might be a common type of just the entities or something.